src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -50,12 +54,13 @@ public void end(boolean interrupted) {
   public boolean isFinished() {
     if(m_ShooterSubsystem.wheelReady()){
       SmartDashboard.putBoolean("wheelReady", true);
-      if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
-        m_ShooterSubsystem.runCargo(true);
+      //if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
+        m_ShooterSubsystem.runCargo(true,true);
+        m_CDSSubsystem.CDSBeltWheelControllerToggle(false);
         i++;
-        if(i==10){ //Expected to add a 200ms delay
+        if(i==100){ //Expected to add a 200ms delay
```

Comment:
No longer 200ms with 100.  Should we add a constant here?

Reply:
I don't know how we could use constants here, I'll make sure to keep the comment up to date until I can rework a new system.